src/presentation/http/router/noteList.ts

```diff
@@ -36,46 +78,22 @@ const NoteListRouter: FastifyPluginCallback<NoteListRouterOptions> = (fastify, o
         'authRequired',
       ],
     },
-    schema: {
-      querystring: {
-        page: {
-          type: 'number',
-          minimum: 1,
-          maximum: 30,
-        },
-      },
+    schema: noteListSchema,
+  }, createNoteListHandler(false));
 
-      response: {
-        '2xx': {
-          description: 'Query notelist',
-          properties: {
-            items: {
-              id: { type: 'string' },
-              content: { type: 'string' },
-              createdAt: { type: 'string' },
-              creatorId: { type: 'string' },
-              updatedAt: { type: 'string' },
-            },
-          },
-        },
-      },
-    },
-  }, async (request, reply) => {
-    const userId = request.userId as number;
-    const page = request.query.page;
-
-    const noteList = await noteService.getNoteListByUserId(userId, page);
-    /**
-     * Wrapping Notelist for public use
-     */
-    const noteListItemsPublic: NotePublic[] = noteList.items.map(definePublicNote);
-
-    const noteListPublic: NoteListPublic = {
-      items: noteListItemsPublic,
+  /**
+   * Get note list created by the user
```

Comment:
I don't think that we need to introduce the separate method for that. Reuse the existing one. You can use get parameters to pass a feed type like "my"